src/shared/lib/getCanonicalMutationType.ts

```diff
@@ -98,6 +98,7 @@ export function getProteinImpactTypeFromCanonical(mutationType:CanonicalMutation
         case "fusion":
         case "silent":
```

Comment:
So, I can add this last one, but I think i should leave other just in case there are others what we don't know about, agreed?